lib/src/command/upgrade.dart

```diff
@@ -36,6 +49,10 @@ class UpgradeCommand extends PubCommand {
         help: 'Precompile executables in immediate dependencies.');
 
     argParser.addFlag('packages-dir', hide: true);
+
+    argParser.addFlag('breaking',
```

Comment:
This will also update the pubspec.yaml even if the change is not breaking.
Maybe `--resolvable` is better. Not sure...

Reply:
Yeah, this seems like a solid PM question :D